projects/packages/videopress/tests/php/VideoPress_Uploader_Test.php

```diff
@@ -172,7 +172,7 @@ public function test_get_upload_token( $response_from_server, $expected, $throw
 			$callback = array( $this, 'return_valid_response' );
 		} elseif ( 'empty' === $response_from_server ) {
 			$callback = array( $this, 'return_empty_response' );
-		} elseif ( 'error' === $response_from_server ) {
+		} else {
 			$callback = array( $this, 'return_wp_error' );
 		}
```

Comment:
Personally I'd probably do like this instead in this case:

```suggestion
		} elseif ( 'error' === $response_from_server ) {
			$callback = array( $this, 'return_wp_error' );
		} else {
			$this->fail( "Unsupported response '$response_from_server'" );
		}
```
Or maybe throw a different exception.